src/GitVersionCore/CoreModule.cs

```diff
@@ -0,0 +1,58 @@
+using System;
+using GitVersion.BuildServers;
+using Microsoft.Extensions.DependencyInjection;
+using GitVersion.Common;
+using GitVersion.Configuration;
+using GitVersion.Logging;
+using Microsoft.Extensions.Options;
+using Environment = GitVersion.Common.Environment;
+
+namespace GitVersion
+{
+    public class CoreModule : IModule
```

Comment:
Isn't `GitVersionModule` easier to distinguish and understand than `CoreModule`?

Reply:
changed to GitVersionModule